app/_src/gateway/get-started/load-balancing.md

```diff
@@ -24,7 +24,31 @@ instead of a specific host. For the purposes of our example, the upstream will p
 
 Here is a diagram illustrating the setup:
 
-![Upstream targets](/assets/images/products/gateway/getting-started-guide/upstream-targets.png)
+<!--vale off-->
+
+{% mermaid %}
+flowchart LR
+  A("`Route 
+  (/mock)`")
+  B("`Service
+  (example_service)`")
+  C(Load balancer)
+  D(httpbin.org)
+  E(httpbun.com)
+  
+  subgraph id1 ["`**KONG GATEWAY**`"]
+    A --> B --> C
+  end
+
+  subgraph id2 ["`Targets (example_upstream)`"]
```

Comment:
Is it possible to have targets appear on top of example_upstream?

Reply:
Unfortunately no.  This is what happens when you try to add a line 
<img width="737" alt="Screenshot 2024-02-28 at 1 00 56 PM" src="https://github.com/Kong/docs.konghq.com/assets/54370747/19425df4-627d-4219-9d35-c2e253708a32">
break to a title: